### PR TITLE
Update finalizer tests to validate against an ingress finalizer

### PR DESCRIPTION
**Pull request by skmatti**
After adding new finalizer (V2 Finalizer), update finalizer upgrade tests to validate against either of v1 or v2 finalizers.

### CODE REVIEW

**Comment by rramkumar1** [comment]
@skmatti Can you update the description a bit to describe what exactly you are doing? To me it seems like you just created a couple helper functions and used them throughout the code but I would like to be sure of that.

**Comment by skmatti** [comment]
> @skmatti Can you update the description a bit to describe what exactly you are doing? To me it seems like you just created a couple helper functions and used them throughout the code but I would like to be sure of that.

The v2-frontend-namer introduced a new finalizer (common.FinalizerKeyV2). Now there are two ingress finalizers after this change. But the finalizer tests were setup to validate only against old finalizer. This PR updates the tests so they expect one of the two ingress finalizers for validation.

File: pkg/e2e/helpers.go [hunk 1]
```diff
@@ -473,3 +475,27 @@ func CheckNegStatus(svc *v1.Service, expectSvcPors []string) (annotations.NegSta
 	}
 	return negStatus, nil
 }
+
+// CheckFinalizer asserts that an ingress finalizer exists on Ingress.
+func CheckFinalizer(ing *v1beta1.Ingress) error {
```
Comment (rramkumar1) [inline pkg/e2e/helpers.go]: This is checking for any finalizer right? If so I would call this CheckForAnyFinalizer().

Comment (skmatti) [reply pkg/e2e/helpers.go]: Done

**Pull request merged**